packages/js/product-editor/src/contexts/validation-context/README.md

```diff
@@ -0,0 +1,80 @@
+# Validations and error handling
+
+This directory contains some components and hooks used for validations in the product editor.
+
+## Add a link to error snackbar
+
+### What happens when there is an error in the form?
+
+1- Fields registered in the validator context [will get validated](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L87-L110). A field can be registered by making use of the useValidation hook.
```

Comment:
```suggestion
1. Fields registered in the validator context [will get validated](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L87-L110). A field can be registered by making use of the useValidation hook.
```
Formatting is off otherwise